solutions/community-deploy/cloud/AWS/README.md

```diff
@@ -59,21 +62,27 @@ key_name            = "devops-key"
 terraform validate && terraform apply -var-file user.tfvars --auto-approve
 ```
 
+**Note:** AWS instance public ipaddress of N-nodes can be selected from the terraform script execution output.
+
 ## Network and Storage Configuration.
 
-- Execute `/home/centos/setup.sh` to setup Network and Storage devices for CORTX. Script will reboot instance on completion. 
+- Execute `/home/centos/setup.sh` on N-node to setup Network and Storage devices for CORTX. Script will reboot instances on completion. 
```

Comment:
can we write `on all nodes in cluster` instead of `N-node` on all instances of changed code?

Reply:
fixed